tutorials/btp-app-prepare-btp/btp-app-prepare-btp.md

```diff
@@ -56,9 +57,9 @@ Continue with **Step 8: Check and assign entitlements** below.
 [ACCORDION-BEGIN [Step 3: ](Create a Live Account)]
 If you choose to create an account on Live, you have to select a number of services that you need to subscribe to, for example, an SAP HANA database. For each service, there are so-called `entitlements`, which are basically the service plans and the number of units that you want from each service. When you create an account, you need to provide these also.
 
-[VALIDATE_1]
 
 
+[VALIDATE_1]
```

Comment:
This code is obsolete. Please use v2 parser syntax.

Reply:
please see comment.